### PR TITLE
Fix path prefix in our notification update pipeline

### PR DESCRIPTION
**Pull request by weshaggard**

### CODE REVIEW

File: eng/pipelines/notifications.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ stages:
           notification-creator `
             --organization $(Organization) `
             --project $(Project) `
-            --path-prefix \"\\$(PathPrefix)\" `
+            --path-prefix "\$(PathPrefix)" `
```
Comment (sima-zhu) [inline eng/pipelines/notifications.yml:61]: What's the motivation of making this change? It is working in my local. Do we have the change for simplicity?

Comment (weshaggard) [reply eng/pipelines/notifications.yml:61]: This escaping is not working in CI as it ends up with extra slashes and then pipelines fail to filter the pipelines. You can see some failures at https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1240423&view=results.  I also had to fix the codeowner parsing with https://github.com/Azure/azure-sdk-tools/pull/2420.

Comment (sima-zhu) [reply eng/pipelines/notifications.yml:61]: Thanks for clarification and I see what you mean! I have reviewed the changes for code owner parsing. 
image

**Pull request merged**